attachments/templates/attachments/list.html

```diff
@@ -9,8 +9,21 @@
             <th scope="col">Remove</th>
         </tr>
     </thead>
-    {% for upload, property_form in session.upload_forms %}
+    {% for error, upload, property_form in session.upload_forms %}
         <tbody class="upload-item">
+        {% if error %}
+            <tr>
+                <td colspan="3">
+                    <div class="alert alert-danger alert-dismissable">
```

Comment:
Not sure how I feel about embedded a whole `alert` div inside a table. What about just putting `danger` on the `tr`, ala https://getbootstrap.com/docs/3.3/css/#tables-contextual-classes